.circleci/base_config.yml

```diff
@@ -12,11 +12,11 @@ executors:
         user: root
   machine-executor:
     machine:
-      docker_layer_caching: false
+      docker_layer_caching: true
```

Comment:
Is this for Nexus or something else? A little confused on why this is in this PR

Reply:
I was just checking to see if it had an effect on the build and test times. It doesn't unfortunately and I forgot to remove it.